src/main/java/frc/robot/commands/SelectByRobotState.java

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import java.util.function.Supplier;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.robotcontainers.CompRobotContainer.RobotState;
+
+public class SelectByRobotState extends CommandBase {
```

Comment:
GamePiece state can be used by the arm command directly.  We don't need a separate command.

Reply:
I suppose we could integrate GamePieceState directly into an ArmCommand

Reply:
Yes, ArmCommand will have a supplier arg of GamePieceState and GripperCommand(intake) will have a consumer arg of GamePieceSate.